scanner_test.go

```diff
@@ -353,6 +353,64 @@ func TestScan(t *testing.T) {
 			),
 		},
 
+		{
+			name: "oct null",
+			text: "\"\\0\"",
+			want: []byte{
+				0x00,
+			},
+		},
+		{
+			name: "oct null 8",
+			text: "\"\\08\"",
+			want: []byte{
+				0x00, 0x38,
+			},
+		},
+		{
+			name: "oct double",
+			text: "\"\\13\"",
+			want: []byte{
+				0x0b,
+			},
+		},
+		{
+			name: "oct double X",
+			text: "\"\\13" + "X\"",
```

Comment:
Is there a particular reason this string is split in twain?

Reply:
Yes. It turns out that Go wants its octal escape sequences to contain exactly three digits. Without the split this is a syntax error.

Reply:
Hmm. I thought there was but then when I tried removing the split everything still worked just fine. So, not sure what that was about but it is fixed now.